lib/create.js

```diff
@@ -53,23 +53,39 @@ var toUnderscoreFromCamel = function(string) {
 };
 
 var Create = {
-  contract: function(directory, name, callback) {
+  contract: function(directory, name, options, callback) {
+    if(typeof options == "function") {
+      callback = options;
+    }
+
     var from = templates.contract.filename;
     var to = path.join(directory, name + ".sol");
 
+    if (!options.force && fs.existsSync(to)) {
+      return callback('Can not create ' + name + '.sol: file exists');
```

Comment:
Ah shoot, sorry to ask for another set of changes - I just looked and other invocations of `callback(err)` actually pass an error object; mind changing these to `callback(new Error('Can not create...'))`?

Reply:
Good catch! I've added the Error object.